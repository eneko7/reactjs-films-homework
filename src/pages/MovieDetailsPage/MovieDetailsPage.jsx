import React from 'react';
import './MovieDetailsPage.scss';
import MoviesGrid from '../../components/MoviesGrid';

const MovieDetailsPage = () => (
  <MoviesGrid />
);

export default MovieDetailsPage;
