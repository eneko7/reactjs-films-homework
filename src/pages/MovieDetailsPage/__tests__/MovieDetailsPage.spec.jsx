import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import MovieDetailsPage from '../index';

const shallow = new ShallowRenderer();

describe('MovieDetailsPage Snapshot', () => {
  test('renders', () => {
    const component = shallow.render(
      <MovieDetailsPage />,
    );
    expect(component).toMatchSnapshot();
  });
});
