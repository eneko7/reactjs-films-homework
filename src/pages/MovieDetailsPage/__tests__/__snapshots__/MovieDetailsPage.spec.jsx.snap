// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetailsPage Snapshot renders 1`] = `
<div
  className="main_container"
>
  <Connect(withRouter(Header)) />
  <Connect(MoviesGrid) />
  <Footer />
</div>
`;
